content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/magnetic_dipole_frequency/index.rst

```diff
@@ -3,29 +3,67 @@
 
-.. topic:: Purpose
+.. Purpose::
 
-    Here, we provide a physical description for the harmonic magnetic dipole.
+    In the frequency domain, we consider harmonic electric and magnetic fields.
+    Here, we provide a physical description of the harmonic magnetic dipole.
     This is used to develop a mathematical expression which can be used to replace the magnetic source term in Maxwell's equations.
 
 
+**General Definition**
 
+The harmonic magnetic dipole can be thought of as an infinitessimally small loop which carries a harmonic current.
+The strength of the source is therefore defined by a harmonic dipole moment :math:`\mathbf{m}(\omega)`.
+For a harmonic magnetic dipole defined by vector surface area :math:`\mathbf{S}` and harmonic current :math:`I(\omega) = I e^{i\omega t}`, the dipole moment is given by:
 
 
-An magnetic dipole is an infinitesimal loop of wire of area :math:`S` carrying a current :math:`I`. The strength of the dipole is given by the magnetic dipole moment (:math:`m`)
+.. figure:: images/H_source_magnetic_dipole.png
+		:align: right
+		:figwidth: 50%
+		:name: MagDipole_freq
+		
+		Physical representation of the magnetic dipole source where :math:`\mathbf{m}` = 1 Am :math:`\!^2`.
+
+
+.. math::
+	\mathbf{m} (\omega) = \mathbf{m} \, e^{i\omega t} = I \mathbf{S} \, e^{i\omega t}
+	:name: m_harmonic_def
+
+where :math:`\mathbf{m} = I \mathbf{S}` is the vector amplitude of the dipole moment.
+When formulating Maxwell's equations in the frequency domain, :math:`e^{i\omega t}` is generally suppressed.
+As a result, the source term for the harmonic magnetic dipole is given by:
+
 
 .. math::
-	m = IS
+	\mathbf{J_m^s} = -i\omega \mu I\mathbf{S} \, \delta (x) \delta (y) \delta (z)
+	:name: Jm_harmonic_def
+
+where :math:`\delta (x)` is the Dirac delta function.
+By including the source term, Maxwell's equations in the frequency domain are given by:
 
-The magnetizaton vector is given by, for a loop laying in the yz plane:
 
 .. math::
-	\mathbf{M} = m \mathbf{u_x} \delta(x) \delta(y) \delta(z)
+	\begin{split}
+	\nabla \times \mathbf{E_m} & + i \omega  \mu \mathbf{H_m} = - i \omega \mu I \mathbf{S} \, \delta(x) \delta(y) \delta(z)  \\
+	&\nabla \times \mathbf{H_m} - ( \sigma + i \omega \varepsilon ) \mathbf{E_m} = 0
+	\end{split}
+	:name: Maxwells_harmonic_Jm
+
+
+where subscripts :math:`_m` remind us that we are considering a magnetic source.
+The source is responsible for generating a primary magnetic field in the surrounding region (:numref:`MagDipole_freq`).
+According to :ref:`Faraday's law<faraday_differential_frequency>`, the harmonic nature of the primary magnetic field generates rotational electric fields.
+In matter, this leads to an induced current density which produces secondary magnetic fields according to the :ref:`Ampere-Maxwell equation<ampere_maxwell_differential_frequency>`.
 
 .. _notebook: https://github.com/geoscixyz/em_apps/blob/master/notebooks/maxwell1_fundamentals/HarmonicDipoleWidget_MD.ipynb
```

Comment:
would you mind updating the link to point to azure notebooks instead? (and keep the name for reference?)